### DISCUSSION TITLE
Increase crons amount limit

### DISCUSSION THREAD

**Discussion by wscourge** [vercel/community#3472] [Idea]
There's hardcoded `20`, that's not much.

**Comment by amyegan** [comment]
Hey @wscourge. The hard limit of 20 cron jobs per project is a [beta](https://vercel.com/docs/concepts/release-phases#beta) phase restriction. You can find more information about cron job limits at https://vercel.com/docs/cron-jobs#limits

**Comment by wscourge** [comment]
OK I see, thank you for pointing me there.
